<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseDetailsDialog</class>
 <widget class="QDialog" name="LicenseDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Generic</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>License Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="licenseTypeLabel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>License Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="licenseKeyLabel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Locked to Hardware ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="licenseHwidLabel"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Features</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QLabel" name="featureNameLabel">
        <property name="text">
         <string notr="true">&lt;feature name&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="featureValueLabel"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
